install/gcp/deployment_manager/istio-cluster.jinja

```diff
@@ -178,7 +178,7 @@ resources:
             {% endif %}
 
             {% if  properties['enableTracing'] %}
-              KIALI_OPTIONS=$KIALI_OPTIONS" --set kiali.dashboard.jaegerURL=http://$(kubectl get svc tracing -n istio-system -o jsonpath='{.spec.clusterIP}'):80"
+              KIALI_OPTIONS=$KIALI_OPTIONS" --set kiali.dashboard.jaegerURL=http://$(kubectl get svc tracing -n istio-system -o jsonpath='{.spec.clusterIP}'):9411"
```

Comment:
Are we sure this is correct? I just installed Istio 1.0.4 and port 80 is what I need to use:

```
$ curl --head http://$(kubectl get svc tracing -n istio-system -o jsonpath='{.spec.clusterIP}'):80
HTTP/1.1 200 OK
Content-Type: text/html; charset=utf-8
Date: Mon, 07 Jan 2019 15:49:33 GMT
Content-Length: 1179
```

Reply:
ok, i get it, that setting points the UI to the frontend exposed IP address...i shoudn't even use 'clusterIP` there anyway.  

See comment above, i'll just remove the configs that setup the integrated `jaeger` stuff since we're not exposing the endpoints